tbx/work/models.py

```diff
@@ -266,83 +265,95 @@ def read_time(self):
 
 
 # Work index page
-@method_decorator(get_default_cache_control_decorator(), name="serve")
 class WorkIndexPage(SocialFields, Page):
-    template = "patterns/pages/work/work_listing.html"
+    template = "patterns/pages/work/work_index_page.html"
 
     subpage_types = ["HistoricalWorkPage", "WorkPage"]
 
-    intro = RichTextField(blank=True)
-
-    @property
+    @cached_property
     def works(self):
-        # Get list of work pages that are descendants of this page
-        work_pages = HistoricalWorkPage.objects.descendant_of(self).live()
-
-        # Order by most recent date first
-        work_pages = work_pages.order_by("-date", "-pk")
+        pages = (
+            self.get_children()
```

Comment:
I'm noticing that when you filter by a tag, it can take a long time for the page to load - the second time you visit that same filter url it is much quicker. I wonder if it is just the image renditions all being created for the first time? I wonder if there is anything we can do on launch to stop it initially being slow?

Reply:
I noticed this as well, I think it's the query itself which needs to be optimised. Needs additional investigation to look at performance optimisation

Reply:
Can you put a ticket in the backlog for this?

Reply:
Sure, I've created one [here](https://torchbox.monday.com/boards/1192293412/pulses/1382356404)